flow-typed/box-ui-elements.js

```diff
@@ -370,6 +370,18 @@ type Collection = {
     totalCount?: number,
 };
 
+type FolderUploadItem = {
```

Comment:
Unless you have time don't have to do in this commit but in your followup:

Since FolderUploadItem and UploadItem are essentially the same, other than a couple poorly labelled and optional.

`UploadFile` ---> `WebkitFileObject`

```js
type UploadItem = {
    api?: PlainUploadAPI | MultiputUploadAPI, // made optional
    item?: BoxItem, // renamed
    error?: Object,
    extension: string,
    fileObject: WebkitFileObject, // renamed, OR nativeFileObject
    name: string,
    progress: number,
    size: number,
    status: UploadStatus,
    options?: UploadItemAPIOptions,
    isFolder?: boolean,
};
```

Reply:
Will merge this and create a new PR. Thanks!